src/MainPage/MainPage.js

```diff
@@ -118,9 +118,6 @@ function MainPage() {
     fetchData(2);
   }, [])
 
-
```

Comment:
Not much to comment on, but I have full trust  in Kyle that this will enhance the repo.